tests/test_aioresponses.py

```diff
@@ -314,3 +314,19 @@ def test_timeout(self, mocked):
 
         with self.assertRaises(asyncio.TimeoutError):
             self.run_async(self.request(self.url))
+
+    @aioresponses()
+    def test_callback(self, m):
+        body = b'New body'
+
+        @asyncio.coroutine
+        async def callback(match, url, *args, **kwargs):
```

Comment:
Maybe callback should be simple method, not coroutine ?

Reply:
Agree, changed to simple function.